cmd/destroy/destroy.go

```diff
@@ -276,14 +299,13 @@ func (dc *destroyCommand) getDestroyer(ctx context.Context, opts *Options) (dest
 			if err := dc.buildCtrl.buildImageIfNecessary(ctx, manifest); err != nil {
 				return nil, err
 			}
-			destroyImage, err = model.ExpandEnv(manifest.Destroy.Image, false)
+			opts.Manifest.Destroy.Image, err = model.ExpandEnv(manifest.Destroy.Image, false)
```

Comment:
```suggestion
                         opts.Manifest = manifest
			opts.Manifest.Destroy.Image, err = model.ExpandEnv(manifest.Destroy.Image, false)
```
if we dont set `opts.manifest = manifest` we will panic:
`````
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x50 pc=0x1027bd0a0]

goroutine 1 [running]:
github.com/okteto/okteto/cmd/destroy.(*destroyCommand).getDestroyer(0x140013bfb90, {0x103a179d0, 0x14000056048}, 0x14001541580)
	github.com/okteto/okteto/cmd/destroy/destroy.go:303 +0x2d0
github.com/okteto/okteto/cmd/destroy.Destroy.func1(0x14001625900?, {0x105441ad0?, 0x0?, 0x0?})
	github.com/okteto/okteto/cmd/destroy/destroy.go:201 +0x7f0
github.com/spf13/cobra.(*Command).execute(0x14001625900, {0x105441ad0, 0x0, 0x0})
	github.com/spf13/cobra@v1.5.0/command.go:872 +0x4d0
github.com/spf13/cobra.(*Command).ExecuteC(0x140015fa500)
	github.com/spf13/cobra@v1.5.0/command.go:990 +0x354
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
	github.com/okteto/okteto/main.go:156 +0x788
`````